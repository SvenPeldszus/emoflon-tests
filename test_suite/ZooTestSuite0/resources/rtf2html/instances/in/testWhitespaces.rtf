{\rtf1 {{ WHITESPACE	TAB
NEWLINE}
{  2WHITESPACE		2TAB

2NEWLINE}}}