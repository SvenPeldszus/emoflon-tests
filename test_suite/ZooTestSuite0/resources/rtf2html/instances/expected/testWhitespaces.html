<span><span style="rtfcommand: rtf1;"><span><span> WHITESPACE	TAB
NEWLINE</span>
<span>  2WHITESPACE		2TAB

2NEWLINE</span></span></span></span>